### PR TITLE
Kob 45044 documents for 4 14 release

### PR DESCRIPTION
**Pull request by tungmhoang**
### Summary

- Documents for 4.14 release

### Related PRs, issues, or features (optional)

- https://kobiton.atlassian.net/browse/KOB-41007. Preview: http://10.2.7.208/device-lab-management/deviceConnect/remote-update-deviceconnect
- https://kobiton.atlassian.net/browse/KOB-45412. Preview: http://10.2.7.208/automation-testing/scripting/obfuscate-private-data-in-appium-script
- https://kobiton.atlassian.net/browse/KOB-45891. Preview: http://10.2.7.208/devices/device-metadata#_lightning_mode
- https://kobiton.atlassian.net/browse/KOB-45892. Preview: http://10.2.7.208/organization/sso-authentication/use-okta

### Metadata

- [x] Adds new file(s)
- [x] Edits existing file(s)
- [ ] Removes file(s)

### PR contributor checklist

- [x] My PR follows the [Kobiton Docs contributor guidelines](https://github.com/kobiton/docs/blob/main/CONTRIBUTE.adoc), meaning:

    - My content contains correct spelling and grammar.
    - My directories and files are [named](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-and-file-names) and [structured](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-structure) correctly.
    - My style and voice follows the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human).
    - I added all new pages to their section's [`nav.adoc` file](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#configure-navigation-in-navadoc).
    - I removed all localizations (like `en-us`) from my URLs.

### CODE REVIEW

File: docs/modules/automation-testing/pages/scripting/obfuscate-private-data-in-appium-script.adoc [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+.Example (JavaScript)
+[source,javascript]
+
+await driver.updateSettings({'kobiton:privateMode': false})
+
+After this command, the data is no longer obfuscated.
+
+== Examples
+
+Below is a full example of a JavaScript code using `wd` that perform a simple login on a website and obfuscates the username and password during login:
```
Comment (erincbailey) [inline docs/modules/automation-testing/pages/scripting/obfuscate-private-data-in-appium-script.adoc:40]: slight revision:
Below is a complete example of JavaScript code using 'wd' that demonstrates a simple login on a website, with the username and password obfuscated during the process.

File: docs/modules/automation-testing/pages/scripting/obfuscate-private-data-in-appium-script.adoc [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+      console.error(`quit driver: ${err}`)
+    }
+  }
+  })
+})
+
+
+----
+
+== Obfuscated data in Session Explorer
```
Comment (erincbailey) [inline docs/modules/automation-testing/pages/scripting/obfuscate-private-data-in-appium-script.adoc:148]: Consider revision: Obfuscated private data in Session Explorer

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+== Prerequisites
+
+* Only available for Cloud and Hybrid customers.
+* Gather the internal IP address or hostname of the Mac mini host(s).
+* Kobiton administrator with the ADMIN predefined permission.
+* Remote update is available in deviceConnect version 4.13.0 and above.
+
+== Perform a remote deviceConnect update manually
+
+When the below notification banner appears in the Portal, a deviceConnect update is available for at least 1 Mac mini host in your organisation:
```
Comment (erincbailey) [inline docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:16]: American spelling has a z: organization

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+image:new-version-available.png[width=1000,alt="The banner about new version of deviceConnect"]
+
+To update deviceConnect remotely, log into the Kobiton Portal with an administrator account.
```
Comment (erincbailey) [inline docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:20]: change "log into" to "log in to"

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+Select the *Device Management* tab.
+
+The *Device Management* page lists all dedicated mobile devices for the org grouped by Mac mini host. Locate the Mac mini host you want to update and select the corresponding *Upgrade* button.
+
+[TIP]
+A Mac mini host that does not have the latest deviceConnect version has a warning icon next to its name in Device Management.
+
+[IMPORTANT]
+Make sure the current deviceConnect version on the Mac mini host is at least 4.13.0 before continuing, as deviceConnect below this version cannot be remotely updated.
```
Comment (erincbailey) [inline docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:32]: from "cannot be remotely updated" to "cannot be updated via the Portal."

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+image:upgrade-host-machine-device-management.png[width=1000,alt="The Upgrade button next to the hosting machine name in Device Management"]
+
+In the confirmation modal, choose *Upgrade* again to proceed.
+
+The Mac mini host will download the latest deviceConnect version and apply the update. Once the update starts, any active sessions on devices hosted by that Mac mini will be terminated. During the update, no actions can be performed on the Mac mini or the connected devices (all the buttons are grayed out).
+
+After the update finished, the devices should automatically come back online. Double-check the current version of deviceConnect on the Mac mini host to verify if the update is successful.
```
Comment (erincbailey) [inline docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:40]: finished should be finishes

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+== Enable or disable automatic deviceConnect update for an organization
+
+By default, automatic deviceConnect update is enabled for an organization. When automatic update is enabled, the system will periodically check for the latest deviceConnect version, download it, then update.
```
Comment (erincbailey) [inline docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:44]: I thought manual was the default? Can you confirm with Ngo

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+The update only starts when all devices hosted by the Mac mini are not in use, i.e. there is no active sessions on the devices. If there are active sessions on the devices, the update is skipped until the next update check (about 30 minutes between each check).
+
+To check the current automatic update setting for your organization, or to enable/disable automatic update, follow the steps below:
+
+Log into the Kobiton Portal with an administrator account.
```
Comment (erincbailey) [inline docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:50]: Log in to

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+include::profile:partial$open-settings.adoc[]
+
+Select the *Device Management* tab. If the organization has 1 least 1 Mac mini host with online devices, the *New version upgrade* section displays and *Automatic* is selected by default:
```
Comment (erincbailey) [inline docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:54]: Small typo - first 1 should be "at" - "If the organization has at least 1..."

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+[WARNING]
+====
+
+When adding a new hosting machine with deviceConnect v4.12 or earlier to an organization, neither the machine nor its devices appear on the Device Management page in the portal.
+
+To avoid this issue, ensure that deviceConnect v4.13 or later is installed on all new hosting machines.
+
+====
+
+When devices are offline due to deviceConnect version being too old, you will see  `Kobiton upgrade required` under the device message in *Settings* -> *Device Management*:
```
Comment (erincbailey) [inline docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:77]: "being too old" to "being outdated"

**Review by erincbailey — CHANGES_REQUESTED** [review]
minor updates

File: docs/modules/automation-testing/pages/scripting/obfuscate-private-data-in-appium-script.adoc (hunk 1 above)
Comment (tungmhoang) [reply docs/modules/automation-testing/pages/scripting/obfuscate-private-data-in-appium-script.adoc:40]: updated

File: docs/modules/automation-testing/pages/scripting/obfuscate-private-data-in-appium-script.adoc (hunk 2 above)
Comment (tungmhoang) [reply docs/modules/automation-testing/pages/scripting/obfuscate-private-data-in-appium-script.adoc:148]: updated

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc (hunk 3 above)
Comment (tungmhoang) [reply docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:16]: updated

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc (hunk 4 above)
Comment (tungmhoang) [reply docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:20]: updated

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc (hunk 5 above)
Comment (tungmhoang) [reply docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:32]: updated

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc (hunk 6 above)
Comment (tungmhoang) [reply docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:40]: updated

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc (hunk 7 above)
Comment (tungmhoang) [reply docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:44]: When I register a new org, the default is Automatic. I will double check again.

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc (hunk 8 above)
Comment (tungmhoang) [reply docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:50]: updated

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc (hunk 9 above)
Comment (tungmhoang) [reply docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:54]: updated

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc (hunk 10 above)
Comment (tungmhoang) [reply docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc:77]: updated

File: docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc (hunk 7 above)
Comment (tungmhoang) [reply docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc]: It is set to Manual by default. I have updated the docs.

**Review by erincbailey — APPROVED** [review]
approved!

**Pull request merged**